webapp/src/shared/types/colors.ts

```diff
@@ -0,0 +1,9 @@
+export type Color =
+  | '--color-warning'
+  | '--color-submit'
+  | '--color-online'
+  | '--color-offline'
+  | '--color-light'
+  | '--color-background'
+  | '--color-dark'
+  | '--color-game';
```

Comment:
I think is strange to have color codes as css variables in tsx code we are sacrifiying legibility just to avoid importing this in a few places and saving a few lines of bundled `js` code. I really love the idea of union types but I think we should use them only when they add real value without substracting

```suggestion
export default enum Color = {
  WARNING =  '--color-warning',
   SUBMIT = '--color-submit',
  ONLINE =  '--color-online',
  OFFLINE = '--color-offline',
  LIGHT = '--color-light',
  BACKGROUND = '--color-background',
  DARK = '--color-dark',
  GAME = '--color-game'
}
```

or, if you really hate enums :smile: we could use a simple `js` object

```ts
export default Colors = {
  WARNING :  '--color-warning',
  SUBMIT : '--color-submit',
  ONLINE :  '--color-online',
  OFFLINE : '--color-offline',
  LIGHT : '--color-light',
  BACKGROUND : '--color-background',
  DARK : '--color-dark',
  GAME : '--color-game'
}
```